### PR TITLE
update to use ses-0.9.1

### LINKED ISSUE #1290: upgrade to SES-0.9.0
Upgrading to yesterday's new SES-0.9.0 release. There's one problem with `String.toLocaleLowerCase`, which was removed in 0.9.0, but is used by `rollup`, which causes our `bundle-source` package to break. I'll add a temporary `patch-package` patch to change rollup to use `toLowerCase` instead. The next SES release will probably bring that back (https://github.com/Agoric/SES-shim/issues/378).

### PR DESCRIPTION
**Pull request by warner**
closes #1290

### CODE REVIEW

File: packages/install-ses/install-ses.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'ses';
 // we need to enable Math.random as a workaround for 'brace-expansion' module
 // (dep chain: cosmic-swingset/ag-solo->temp->glob->minimatch->brace-expansion)
 // we need Date.now to build a timer device for cosmic-swingset
-lockdown({ mathTaming: 'unsafe', dateTaming: 'unsafe' });
+lockdown({ mathTaming: 'unsafe', dateTaming: 'unsafe', errorTaming: 'unsafe' });
```
Comment (erights) [inline packages/install-ses/install-ses.js:8]: good

**Review by erights — APPROVED** [review]
LGTM

**Review by erights — COMMENTED** [review]
@kriskowal @warner 

Now that https://github.com/Agoric/SES-shim/pull/382 is merged before this one, any chance we could 
   * skip the current state of this one, 
   * do a minor SES release, perhaps 0.9.1 ? 
   * revise this one to use ses-0.9.1, 
   * and omit the patching of rollup?

**Comment by warner** [comment]
yeah, I'll take my time landing this one, if y'all get an 0.9.1 out first, I'll remove the `patch-package` bits

**Comment by warner** [comment]
new branch omits the `patch-package` change in favor of SES-0.9.1

**Pull request merged**